scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/Scheduler.java

```diff
@@ -637,6 +637,9 @@ boolean killTask(String jobId, String taskName) throws NotConnectedException, Un
     boolean restartTask(String jobId, String taskName, int restartDelay) throws NotConnectedException,
             UnknownJobException, UnknownTaskException, PermissionException;
 
+    boolean restartInErrorTask(String jobId, String taskName) throws NotConnectedException,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: org.ow2.proactive.scheduler.common.exception.NotConnectedException, org.ow2.proactive.scheduler.common.exception.UnknownJobException, org.ow2.proactive.scheduler.common.exception.UnknownTaskException, org.ow2.proactive.scheduler.common.exception.PermissionException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1160)